t/t-batch-transfer.sh

```diff
@@ -199,10 +199,32 @@ assert_ssh_transfer_sessions() {
     (( ++max_expected_start ))
   fi
 
+  # Versions of Git prior to 2.11.0 invoke Git LFS via the "smudge" filter
+  # rather than the "process" filter, so a separate Git LFS process runs for
+  # each downloaded object and spawns its own control socket SSH connection.
+  if [ "download" = "$direction" ]; then
+    gitversion="$(git version | cut -d" " -f3)"
+    set +e
+    compare_version "$gitversion" '2.11.0'
```

Comment:
Git 2.11 is a few years old. I wonder if we should require a minimum Git version for Git LFS to get rid of a few of those version checks?

Reply:
That's a good question—it's probably worth exploring the idea.  I think we'd want to announce any such plans in an issue, with a proposed future release version at the first one where we'd set a new minimum Git version, and collect input from users as to what that minimum should be.  Some folks are likely to be running Git LFS in systems with fairly old Git clients, especially if they have one of the older OS platforms we still support.

Right now, we [claim](https://github.com/git-lfs/git-lfs/blob/f979a7db3b11d77975bc241f2c5ae71b02457d2c/README.md?plain=1#L143) in our `README` that Git v1.8.2 or v1.8.5 is the minimum, although our Git v2.0.0 is the earliest one we [build](https://github.com/git-lfs/git-lfs/blob/f979a7db3b11d77975bc241f2c5ae71b02457d2c/.github/workflows/ci.yml#L166) for use in our CI jobs.

For now and for the near-term release we have planned, though, I don't think we should change anything about the Git versions we try to support.  (As one example of how our project is expected to work with quite old software, when we overlooked the fact that upgrading to Go 1.21 in our 3.5.x releases meant that we dropped support for Windows 7 and 8, that caused [problems](https://github.com/git-for-windows/git/issues/4996) for the Git for Windows project maintainers.)

Reply:
PR #5921 will at least update the `README.md` file to match the minimal Git version we use in our CI jobs.